x/evm/types/types.go

```diff
@@ -1000,8 +1000,8 @@ func (m EventContractCall) ValidateBasic() error {
 		return sdkerrors.Wrap(err, "invalid destination chain")
 	}
 
-	if !common.IsHexAddress(m.ContractAddress) {
-		return fmt.Errorf("invalid contract address")
+	if err := utils.ValidateString(m.ContractAddress); err != nil {
```

Comment:
why these changes?

Reply:
actually, cosmwasm addresses can be longer https://www.mintscan.io/juno/wasm/contract/juno1u45shlp0q4gcckvsj06ss4xuvsu0z24a0d0vr9ce6r24pht4e5xq7q995n

maybe we can default to 128 as the address limit?

Reply:
contract address is 32 bytes, it's 59 + prefix after bech32. I would say 80 is safe, but 128 is a good number.